src/js/lib/play-billing.js

```diff
@@ -166,6 +167,9 @@ export class PlayBillingService {
         supportedMethods: this.serviceURL,
         data: {
           sku: sku.itemId,
+          oldSku: oldPurchase?.itemId,
+          purchaseToken: oldPurchase?.purchaseToken,
+          prorationmode: 'immediateAndChargeProratedPrice',
```

Comment:
Can we add a document comment on what the other options are for prorationmode? (Or at least a link to the documentation?)

Reply:
Yes, good call. I will add a comment.

Reply:
DONE